google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/Table.java

```diff
@@ -152,6 +158,10 @@ public List<ColumnFamily> getColumnFamilies() {
     return columnFamilies;
   }
 
+  public boolean isProtected() {
```

Comment:
nit: isDeletionProtected

Reply:
This was to be consistent with go - https://github.com/googleapis/google-cloud-go/blob/main/bigtable/bttest/inmem.go#L1330. LMK your thoughts